src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -46,13 +46,11 @@ internal class WinHttpHandler : HttpMessageHandler
     public class WinHttpHandler : HttpMessageHandler
 #endif
     {
-#if NET46
+        // These are normally defined already in System.Net.Primitives as part of the HttpVersion type.
+        // However, these are not part of 'netstandard'. WinHttpHandler currently builds against
+        // 'netstandard' so we need to add these definitions here.
         internal static readonly Version HttpVersion20 = new Version(2, 0);
```

Comment:
reviewer note: I will look into removing these in a future PR. We currently build WinHttpHandler for both 'netstandard' and 'netfx' (as a separate NuGet package). But this can be changed so remove the 'netstandard' build and use 'netcoreapp' instead (similar to System.Net.Http builds)